corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.16 on 2023-02-03 06:57
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq import privileges
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_view_app_diff_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        privileges.VIEW_APP_DIFF,
+        skip_edition='Paused,Community,Standard',
```

Comment:
I don't have much experience with the plans yet, but should we not include the `pro`plan here as well?

Reply:
If this is for Standard Plan, shouldn't `skip_edition` be `'Paused,Community'`?

Reply:
@kaapstorm @SmittieC 

Sorry my bad on Plan confusion, this is supposed to be under Advanced Plan, so I have updated it as such